stigs/system/ksp-block-stig-psql-v-214082.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+# https://www.stigviewer.com/stig/postgresql_9.x/2021-06-04/finding/V-214082
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-stig-psql-v-214082
+  namespace: default # Change your namespace
+spec:
+  tags: ["STIG", "PSQL"]
+  message: "Alert! Configs file access has been denied."
+  selector:
+    matchLabels:
+      app: postgres  # Change your matchLabels
+  file:
+    severity: 4
+    matchPaths:
+    - path: /var/lib/postgresql/data/postgresql.conf    #configuration files
```

Comment:
This location seems to be dynamic, for the system I'm testing there is no such location
 - For me, this location is at `/etc/postgresql/13/main/postgresql.conf`

Also, from this [article](https://www.postgresql.org/docs/9.1/storage-file-layout.html), I can conclude that `postgresql.conf` was stored in `/var/lib/pgsql/data` directory for PostgreSQL 8.0 and later

Reply:
Thank you for your review and appropriate changes are done.